src/main/kotlin/org/openrewrite/RecipeMarkdownGenerator.kt

```diff
@@ -647,14 +647,35 @@ class RecipeMarkdownGenerator : Runnable {
                 }
 
                 if (recipes.isNotEmpty()) {
```

Comment:
This conditional should maybe move down, and another added to not show the `Composite Recipes` header if there are no composite recipes.

Reply:
Ahh yep -- another good catch. Will fix.